homeassistant/components/mqtt/client.py

```diff
@@ -117,7 +116,7 @@
 
 MAX_PACKETS_TO_READ = 500
 
-type SocketType = socket.socket | ssl.SSLSocket | mqtt.WebsocketWrapper | Any
+type SocketType = socket.socket | ssl.SSLSocket | mqtt._WebsocketWrapper | Any  # noqa: SLF001
```

Comment:
This seems needed still, but the `WebsocketWrapper` class was made private (renamed)